packages/ui/src/BooleanField.tsx

```diff
@@ -117,7 +115,8 @@ export const BooleanField = ({
           </View>
         </TouchableWithoutFeedback>
       </View>
-      {!interaction && disabledHelperText && <Text size="md">{disabledHelperText}</Text>}
+      {!interaction && disabledHelperText && <FieldHelperText text={disabledHelperText} />}
+      {Boolean(helperText) && <FieldHelperText text={helperText!} />}
```

Comment:
![Screenshot 2024-07-14 at 9 47 08 PM](https://github.com/user-attachments/assets/48bc9be3-6fd9-469a-8f64-0e81a19bc896)

do we want the helper text to hide when the disabled helper text appears? right now both show up at the same time if the field is disabled. That might make sense just double checking! 
![Screenshot 2024-07-14 at 9 47 08 PM](https://github.com/user-attachments/assets/a39f4025-4715-4bc1-9038-5eb48265778e)

Reply:
Yeah I think we do, the helper text will give some context about the field and the disabled text will add different info.

Reply:
Makes sense to me!